main.go

```diff
@@ -56,8 +56,8 @@ func CreateHandler(conf Config, git Git, pullRequests PullRequests, repositories
 	}
 }
 
-// startsWithPlusOne matches strings that start with either a +1 (not followed by other digits) or a :+1: emoji
-var startsWithPlusOne = regexp.MustCompile(`^(:\+1:|\+1($|\D))`)
+// isPlusOneComment matches strings that contain either a +1 (not followed by other digits) or a :+1: emoji
+var isPlusOneComment = regexp.MustCompile(`\+1($|\D)`)
```

Comment:
I was thinking of doing it as a part of this PR, but to think about it, I've changed the tests quite a lot on top of this PR already, so it's probably best to defer it for now, yes.